README.md

```diff
@@ -106,13 +106,21 @@ configure a different parameter. For each enable "i" the bitfield contains:
 
 ## Build Notes
 
+### AtmelStudio
+
 This firmware is written for use with AVR GCC 5.4 and AtmelStudio 7. To compile
 the firmware in Atmel Studio, create a new project and add the .c files in the
 `src` directory to the project. The project Toolchain settings must be modified
 to include the `include` directory containing the firmware header files. With
 the project configured correctly it should be possible to build and debug using
 tools provided by Atmel Studio.
 
+### CMake
+
+Alternatively the firmware can be built using [CMake](https://cmake.org)
```

Comment:
Edit readme to describe a general process for what a user would do with the CMake file and how to use it (commands, etc).